packages/appium/lib/cli/driver-command.js

```diff
@@ -30,18 +32,40 @@ export default class DriverCommand extends ExtensionCommand {
     });
   }
 
+  /**
+   * Uninstall a driver
+   *
+   * @param {DriverUninstallOpts} opts
+   * @return {Promise<ExtRecord<DriverType>>}
+   */
   async uninstall({driver}) {
     return await super._uninstall({installSpec: driver});
   }
 
+  /**
+   * Update a driver
+   *
+   * @param {DriverUpdateOpts} opts
+   * @return {Promise<import('./extension-command.js').ExtensionUpdateResult>}
```

Comment:
```suggestion
   * @return {Promise<import('./extension-command').ExtensionUpdateResult>}
```

You should omit file extension, as sometimes this can confuse the declaration output of TS.

(Please apply this to all your changes)